src/components/directorybrowser/directorybrowser.js

```diff
@@ -1,51 +1,148 @@
+                html += '<a is="emby-linkbutton" class="button-link" href="https://web.archive.org/web/20181216120305/https://github.com/MediaBrowser/Wiki/wiki/Synology-:-Setting-Up-Your-Media-Library-Share" target="_blank">';
+                html += Globalize.translate("LearnHowToCreateSynologyShares");
+                html += "</a>";
+            }  else if("bsd" === systemInfo.OperatingSystem.toLowerCase()) {
+                html += "<br/>";
+                html += "<br/>";
+                html += Globalize.translate("MessageDirectoryPickerBSDInstruction");
+                html += "<br/>";
+                html += '<a is="emby-linkbutton" class="button-link" href="http://doc.freenas.org/9.3/freenas_jails.html#add-storage" target="_blank">';
+                html += Globalize.translate("ButtonMoreInformation");
+                html += "</a>";
+            } else if("linux" === systemInfo.OperatingSystem.toLowerCase()) {
+                html += "<br/>";
+                html += "<br/>";
+                html += Globalize.translate("MessageDirectoryPickerLinuxInstruction");
+                html += "<br/>";
+            }
+            html += "</div>"
         }
-        html += '<form style="margin:auto;">', html += '<div class="inputContainer" style="display: flex; align-items: center;">', html += '<div style="flex-grow:1;">';
-        var labelKey = !0 !== options.includeFiles ? "LabelFolder" : "LabelPath",
-            readOnlyAttribute = options.pathReadOnly ? " readonly" : "";
-        return html += '<input is="emby-input" id="txtDirectoryPickerPath" type="text" required="required" ' + readOnlyAttribute + ' label="' + Globalize.translate(labelKey) + '"/>', html += "</div>", readOnlyAttribute || (html += '<button type="button" is="paper-icon-button-light" class="btnRefreshDirectories emby-input-iconbutton" title="' + Globalize.translate("ButtonRefresh") + '"><i class="md-icon">search</i></button>'), html += "</div>", readOnlyAttribute || (html += '<div class="results paperList" style="max-height: 200px; overflow-y: auto;"></div>'), options.enableNetworkSharePath && (html += '<div class="inputContainer" style="margin-top:2em;">', html += '<input is="emby-input" id="txtNetworkPath" type="text" label="' + Globalize.translate("LabelOptionalNetworkPath") + '"/>', html += '<div class="fieldDescription">', html += Globalize.translate("LabelOptionalNetworkPathHelp"), html += "</div>", html += "</div>"), html += '<div class="formDialogFooter">', html += '<button is="emby-button" type="submit" class="raised button-submit block formDialogFooterItem">' + Globalize.translate("ButtonOk") + "</button>", html += "</div>", html += "</form>", html += "</div>", html += "</div>", html += "</div>"
+        html += '<form style="margin:auto;">';
+        html += '<div class="inputContainer" style="display: flex; align-items: center;">';
+        html += '<div style="flex-grow:1;">';
+        var labelKey;
+        if(options.includeFiles !== true) {
```

Comment:
```suggestion
        if (!options.includeFiles) {
```

Reply:
I'm not sure I got the point. So what if `options.includeFiles` is something that evaluates to `false` but is not `false`?.. We can treat it as `false` as far as I can see. But that could be cleaned up later anyway.